Lib9c/Model/Item/Consumable.cs

```diff
@@ -31,7 +31,10 @@ public Consumable(Dictionary serialized) : base(serialized)
         public override IValue Serialize() =>
             new Dictionary(new Dictionary<IKey, IValue>
             {
-                [(Text) "stats"] = new List(Stats.Select(s => s.Serialize())),
+                [(Text) "stats"] = new List(Stats
+                    .OrderBy(i => i.StatType)
```

Comment:
Is any `IComparer<StatType>` defined?

Reply:
i found this. https://docs.microsoft.com/ko-kr/dotnet/api/system.icomparable?view=netcore-2.0